autoprotocol/protocol.py

```diff
@@ -5877,6 +5886,10 @@ def _refify(self, op_data):
             return self._refify(op_data._as_AST())
         elif isinstance(op_data, Ref):
             return op_data.opts
+        elif isinstance(op_data, Compound):
+            return op_data.InChI
+        elif isinstance(op_data, Informatics):
+            return op_data.as_dict()
```

Comment:
this doesn't work currently... we may need to do `self._refify(op_data.as_dict()`. 
We should add a test to _refify to verify this.

Reply:
oops. sorry about that! will add test for this.